app/views/pages/hiring_content.html.haml

```diff
@@ -0,0 +1,145 @@
+        Our online platform is centred around self-directed learning and volunteer mentoring, and has nearly one million members.
+        Over the next 5 years, we want to become the defacto programming education platform that anyone, anywhere can use for free.
+        Through our work we aim to improve the global standard of programming and increase participation in tech from under-represented groups.
+        You can learn more about Exercism in our #{link_to '"About Exercism" pages', 'https://exercism.org/about', target: '_blank', rel: :noopener}.
+      %p.text-p-xlarge.mb-16
+        At the heart of Exercism is our content - thousands of programming exercises across over 5o programming languages, designed to teach and enable practice of programming concepts, nearly all of which has been written by our volunteer community.
+        As we continue to grow our content, and plan to expand it beyond our traditional exercise format, we need someone to lead our content strategy.
+        This is a hands-on role, where you'll be writing and editor content, alongside planning and leading our volunteer maintainers.
+        Currently this role is split between the CEO and Head of Open Source, both of whom you'll work closely with.
+      %p.text-p-xlarge.mb-32
+        We are building an exceptional team of ambitious, passionate people who love their work, excel at what they do, and have fun doing it.
+        The values of empathy, integrity and personal-growth are core to everything we do.
+        If that sounds exciting to be part of, and you feel you have the skills to help, we'd love for you to apply.
+
+      %h2.text-h2.mb-12 Responsibilities
+      %p.text-p-xlarge.mb-16
+        Your role is to fundmentally take all the work currently done by the CEO and the Head of Open Source,
+        Your role splits roughly into three areas.
+      %p.text-p-xlarge.mb-12
+        The first is strategic, which has the following key responsibilities:
+      %ul.text-p-xlarge.mb-16
+        %li Work with our CEO, Head of Open Source to more tighly define our our wider content strategy for the next three years.
+        %li Work with our senior maintainers to design and document systems that improve the flow and quality of content.
+        %li Work with our content partners (e.g. Packt Publishing) to determine synergies between their content and ours, and how we can make the most use of their educational portfolio.
+      %p.text-p-xlarge.mb-12
+        The second is a leadership role:
+      %ul.text-p-xlarge.mb-16
+        %li You will be the senior content voice for our community, answering questions, arbitrating discussions, and acting as the knowledgable sage of wisdom. This doesn't mean you have to be the most knowledgable on any given subject, but means you must speak to Exercism's perspective on content, education, and how we make pragmatic decisions.
```

Comment:
I would think that you do not speak to a perspective, but you speak from a perspective.  Speaking to a perspective seems unhelpful.

Reply:
To "speak to" can mean to speak on a topic. See the second defintion [here](https://idioms.thefreedictionary.com/speak+to). I've always seen it used to mean to speak about a topic with authority (see this comment [here](https://grammarist.com/usage/speak-to/#comment-2359) for an example of how I'm used to it being used). There's also a [SlackExchange](https://english.stackexchange.com/questions/392331/why-do-so-many-people-speak-to-topics-rather-than-on-or-about-them-is-thi) discussion here on it. 

It's a pretty common idiom in my world 🙂

Reply:
Yeah, it is unfortunate.  In my world, when you are spoken to, you are seldom heard, and so speaking to a subject implies not really communicating.  And yes, the responses in your second "here" link, that is pretty much how I reacted when I started to hear people say "Let me talk to that subject" only 25 years ago, I  was hoping it was not going to  become prevalent, and indeed, I had all but only occasionally heard it in the last 5 years or so.

Also, because "can mean" probably means we can be clearer.